code/drasil-example/Drasil/Projectile/ProblemDescription.txt

```diff
@@ -0,0 +1,44 @@
+﻿Physical System Description
+
+Elements:
+	1) The object
+	2) The ground
+
+Assumptions
+1) The projectile motion is in 2D.
+2) The object is a point mass.
+3) The object's path starts and ends at ground level.
+4) The effects of drag, wind, and the Earth's curvature are ignored.
```

Comment:
Make your assumptions atomic - that is, you should split each of these into its own assumption

Reply:
Should I do the same for A3 (ie. "The object's path starts at ground level." and "The object's path ends at ground level."), or are they related enough that they can be grouped?

Reply:
Thinking about A3, I think you do have one assumption, but ground level isn't really part of it.  Your assumption is that the object and the target are at the same height.  The notion of ground isn't really necessary.  It could be one aircraft launching a cannon at another aircraft at the same height.  In this way you only have one assumption.

You might actually choose to make the height of the projectile launcher and the target inputs.  If the heights input are the same, then you have met this assumption, but allowing the heights to differ, you wouldn't need this assumption at all.  This would be an example of a requirements level variability.

For now, I suggest just changing the assumption as I mentioned above - the launcher and the target are at the same height.  (You will later introduce the modelling convenience that this height is zero.)

Reply:
@smiths I was thinking of adding this as a variability, but right now I just want to get this instance completely working first. If I update this assumption to be that the launch and target heights are equal (for now), should I also remove the concept of ground from the system?

Reply:
I'll jump in here just to say that part of my motivation for suggesting adding ground was because having the object as the only element of the physical system seemed incomplete. I agree with @smiths that saying "ground" is unnecessarily restrictive. So I would replace the "ground" element of the physical system with "target". But maybe @smiths disagrees.

Reply:
Yes, I think we can remove the concept of ground.  It isn't like you are modelling the ground.  It is just the name that you give the height of zero.  You could introduce ground under terminology, if you think it is a helpful concept.

Reply:
Yes, the target should be part of the physical system.  Thank you @bmaclach.  I've been modelling the problem in my head as "launcher" and "target," but we need to make these concepts explicitly part of the documentation.

Reply:
Should we still include the launcher? For example, the object could be "launched" by something being thrown at it, not necessary being "launched" in the traditional sense. If so, does "launch site" make sense as a component of the system? @smiths @bmaclach

Reply:
I like the idea of "launch site" as an element of the physical system.

Reply:
The notion of launcher is abstract.  How it happens doesn't matter.  I think we want parallelism between our concepts of launcher and target.  If we say launch site, then we should say target site, and then it seems like we are adding redundant terminology with the word site.   I think I prefer launcher and target, unless there is a synonym for launcher that sounds better.